README.md

```diff
@@ -35,3 +35,10 @@ The repository is organized this way:
 
```

Comment:
You should probably remove the line above this one, since you've removed the root `providers.json` symlink.

Reply:
Not sure which line you are referring to, anyway I updated the README, thanks for noticing!